Dido's project/CLKW208MangalGoldEdition.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CLK W208 Mangal Gold Edition</title>
+    <link rel="stylesheet" href="./StylesIMG1.css" />
+  </head>
+  <body>
+    <main>
+      <header class="Header">
+        <div class="Text1">
+          <h1>CLK W208 Mangal Gold Edition</h1>
+        </div>
+        <div class="CLK2">
+          <img src="./img/CLK2.jpg" height="400vh" />
```

Comment:
The same things from the other [page ](https://github.com/JavaScript-HTML-CSS-beginner-to-advance/initial-html-repo/pull/2/files#diff-58be2a9c290b182e10139320649cb1a8fb6534d77f010dfba812908015b7ab9a)apply here.